src/org/purefn/starman/jedis.clj

```diff
@@ -129,12 +184,14 @@
                  ::port
                  ::max-total
                  ::max-retries
-                 ::busy-delay-ms]} config]
+                 ::busy-delay-ms
+                 ::namespaces]} config]
```

Comment:
This would be a great opportunity to add a docstring to this constructor function that explains what `config` is.